client/src/app/shared/components/projector/projector.component.ts

```diff
@@ -208,6 +198,37 @@ export class ProjectorComponent extends BaseComponent implements OnDestroy {
         this.updateCSS();
     }
 
+    private updateProjectionSubscription(): void {
+        if (this.projectionSubscription) {
+            this.projectionSubscription.unsubscribe();
+            this.projectionSubscription = null;
+        }
+        this.updateSlides();
+        if (this.projector) {
+            this.projectionSubscription = this.projectionRepo.getGeneralViewModelObservable().subscribe(projection => {
+                if (projection?.current_projector_id === this.projector.id) {
+                    this.updateSlides();
+                }
+            });
+        }
+    }
+
+    private updateSlides(): void {
+        this.slides = (this.projector?.current_projections || []).map(projection => {
+            const slideData: SlideData<any> = {
+                collection: collectionFromFqid(projection.content_object_id),
+                data: projection.content,
+                stable: !!projection.stable,
+                type: projection.type || '',
+                options: projection.options || {}
+            };
+            if (projection.content?.error) {
+                slideData.error = projection.content.error;
+            }
+            return slideData;
+        });
+    }
+
```

Comment:
This would be way easier (or in fact not necessary at all) if `ViewProjector.currentProjections` was `Observable<ViewProjection[]>`

Reply:
I had some thoughts about that already a year ago. If you are interested, we can talk monday or so.